### PR TITLE
Links&Displaying on phones

### PR DESCRIPTION
**Pull request by msynowski**
Fixed links in design and exhibition. Main page looks now good on phones and iPads. Fix of #62 and #63

### CODE REVIEW

File: CNAME [hunk 1]
```diff
@@ -1 +1,4 @@
+<<<<<<< HEAD
 2012.fossasia.org
+=======
+>>>>>>> gh-pages
```
Comment (harshit98) [inline CNAME:4]: Update your branch. Looks like you have not updated your branch.

File: stylesheets/sidebar.css [hunk 2]
```diff
@@ -28,31 +21,94 @@
...
-		float: right;
+       float: right;
 	}
+    #content {
+	margin-top: 10% !important;
+<<<<<<< HEAD
+  	width: 100% !important;
+=======
+
+>>>>>>> gh-pages
```
Comment (harshit98) [inline stylesheets/sidebar.css:36]: Same here. Remove `<<<HEAD` and `>>>gh-pages`.

File: CNAME [hunk 3]
```diff
@@ -1 +1,4 @@
+<<<<<<< HEAD
```
Comment (akshatnitd) [inline CNAME:1]: Remove ```HEAD``` and ```gh-pages```

File: stylesheets/sidebar.css [hunk 4]
```diff
@@ -28,31 +21,94 @@
...
 		margin-top: 5%;
-		background-color: #EEE;
 		border-radius: 10px;
 		width: 100%;
-		float: right;
+       float: right;
 	}
+    #content {
+	margin-top: 10% !important;
+<<<<<<< HEAD
```
Comment (akshatnitd) [inline stylesheets/sidebar.css:32]: Remove ```HEAD``` and ```gh-pages```

File: stylesheets/sidebar.css [hunk 5]
```diff
@@ -28,31 +21,94 @@
...
+=======
+
+>>>>>>> gh-pages
+}
 }
 
+
 @media screen and (min-width: 760px)  and (max-width: 900px){
+    		#side-bar {
+					display: block;
```
Comment (akshatnitd) [inline stylesheets/sidebar.css:43]: Correct the identation

File: stylesheets/sidebar.css [hunk 6]
```diff
@@ -28,31 +21,94 @@
...
 	}
+    #content {
+	margin-top: 10% !important;
+  	width: 60% !important;
+  	float: left !important
+}
 }
+@media only screen and (min-device-width: 481px) and (max-device-width: 1024px) and (orientation:portrait) {
+#side-bar {
+					display: block;
```
Comment (akshatnitd) [inline stylesheets/sidebar.css:61]: Check Indentation

File: stylesheets/sidebar.css [hunk 7]
```diff
@@ -28,31 +21,94 @@
...
+  	width: 100% !important;
+}
+    
+}
+
+
 
 @media screen and (min-width: 900px)  and (max-width: 1100px){
+    		#side-bar {
+					display: block;
```
Comment (akshatnitd) [inline stylesheets/sidebar.css:78]: Check indentation

**Review by akshatnitd — CHANGES_REQUESTED** [review]
Correct the indentation and check inline comments

File: stylesheets/sidebar.css [hunk 8]
```diff
@@ -28,31 +21,94 @@
 
 @media screen and (min-width: 20px) and (max-width : 760px) {
 	#side-bar {
-		display: block;
+        display: block;
```
Comment (anshumanv) [inline stylesheets/sidebar.css:24]: Indentation

File: stylesheets/sidebar.css [hunk 9]
```diff
@@ -28,31 +21,94 @@
...
 		margin-top: 5%;
-		background-color: #EEE;
 		border-radius: 10px;
 		width: 100%;
-		float: right;
+       float: right;
```
Comment (anshumanv) [inline stylesheets/sidebar.css:28]: Indent

File: stylesheets/sidebar.css [hunk 10]
```diff
@@ -28,31 +21,94 @@
...
+					right: -24px;
+		}
 	#side-bar h1 {
 		font-size: 0.7em;
 	}
+    #content {
+	margin-top: 10% !important;
+  	width: 60% !important;
+  	float: left !important
+}
```
Comment (anshumanv) [inline stylesheets/sidebar.css:57]: Indent

**Review by anshumanv — CHANGES_REQUESTED** [review]
Please fix all indentations.

**Comment by mariobehling** [comment]
This is not bringing the project forward directly. What we really need is a proper top menu bar as at fossasia.org.

**Comment by mariobehling** [comment]
Closing this due to inactivity. Please make a new PR, if this issue persists.

**Pull request closed without merging**